### DISCUSSION TITLE
display error | WSL ubuntu20.04

### DISCUSSION THREAD

**Discussion by EsqerYasen** [JanDeDobbeleer/oh-my-posh#5199] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What would you like to see changed/added?

This could change in the documentation!
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/36286533/272b3d39-0c01-4307-bdcf-5790a44ed18c)

**Comment by EsqerYasen** [comment]
i installed the font's , dosnt help. 
oh-my-posh font install meslo

**Reply by JanDeDobbeleer** [reply]
@EsqerYasen you need to configure the font in your terminal as well. This is mentioned in the docs too.

**Comment by EsqerYasen** [comment]
eval "$(oh-my-posh init bash --config /root/themes/cloud-native-azure.omp.json)"
